vmdb/app/views/layouts/_ae_resolve_options.html.haml

```diff
@@ -33,6 +33,38 @@
                          resolve[:new][:object_request],
                          :maxlength         => MAX_NAME_LEN,
                          "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
+=======
```

Comment:
Doh!   I thought I did that, but when I went back and searched I couldn't
find it... Oh well.
On Mar 6, 2015 8:35 PM, "Keenan Brock" <notifications@github.com> wrote:

> In vmdb/app/views/layouts/_ae_resolve_options.html.haml
> <https://github.com/ManageIQ/manageiq/pull/1840#discussion_r25991307>:
>
> > @@ -33,6 +33,38 @@
> >                           resolve[:new][:object_request],
> >                           :maxlength         => MAX_NAME_LEN,
> >                           "data-miq_observe" => {:interval => '.5', :url => url}.to_json)
> > +=======
>
> oops - can you take a look at this.
> Looks like a rebase gone bad
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/1840/files#r25991307>.
>